substra/cli/interface.py

```diff
@@ -1068,7 +1076,7 @@ def update_data_sample(ctx, data_samples, dataset_key):
     - keys: list of data sample keys
     """
     client = get_client(ctx.obj)
-    res = client.link_dataset_with_data_samples(dataset_key, data_samples['keys'])
+    res = client.link_dataset_with_data_samples(dataset_key, load_data_samples_keys(data_samples))
```

Comment:
There is no "--data-samples-path" option for this command, the error message raises by `load_data_samples_keys` would look very wrong.

Reply:
Indeed, but there is an argument called 'data_samples', so we're supposed to get this object, because it is required

Reply:
Yes absolutely, we just need the proper error message.